test/validation/api/traffic_mngr/traffic_mngr.c

```diff
@@ -1677,7 +1677,7 @@ static void dump_tm_subtree(tm_node_desc_t *node_desc)
 	rc = odp_tm_node_info(node_desc->node, &node_info);
 	if (rc != 0) {
 		ODPH_ERR("odp_tm_node_info failed for tm_node=0x%" PRIX64 "\n",
-			 node_desc->node);
+			 odp_tm_node_to_u64(node_desc->node));
```

Comment:
In addition to node_to_u64(), all new conversion functions should be used at least once (dummy usage if not really needed by the test suite). Validation tests need to in minimum verify that those functions are accessible from an application.

Reply:
Fixed in v2